services/github/github-helpers.js

```diff
@@ -11,7 +11,7 @@ const documentation = `
 `
 
 function stateColor(s) {
-  return { open: '2cbe4e', closed: 'cb2431', merged: '6f42c1' }[s]
+  return { open: '2cbe4e', closed: '6f42c1', merged: '6f42c1' }[s]
```

Comment:
A closed (but not merged) PR is still presented as red. I think we need to split this into 2 functions

```js
function issueStateColor(s) {
  return { open: '2cbe4e', closed: '6f42c1' }[s]
}

function pullRequestStateColor(s) {
    return { open: '2cbe4e', closed: 'cb2431', merged: '6f42c1' }[s]
}
```

Reply:
good point, will update